.github/dependabot.yml

```diff
@@ -16,7 +16,7 @@ updates:
   allow:
   - dependency-type: "direct"
   versioning-strategy: "increase"
-  target-branch: v15-branch
+  target-branch: v12-branch
```

Comment:
Do you mean beyond v13 or v12? 

The PR has v12 as the latest version it is tracking.